library/general/uniqueingresshost/src.rego

```diff
@@ -0,0 +1,31 @@
+package k8suniqueingresshost
+
+identical(obj, review) {
+  obj.metadata.namespace == review.object.metadata.namespace
+  obj.metadata.name == review.object.metadata.name
+}
+
+make_apiversion(kind) = apiVersion {
+  g := kind.group
+  v := kind.version
+  g != ""
+  apiVersion = sprintf("%v/%v", [g, v])
```

Comment:
Nit: `:=` instead of `=` here. Same comment applies to a few locations below as well.